index.js

```diff
@@ -179,3 +171,4 @@ var verifier = module.exports = function(cert_url, signature, requestBody, callb
 
 // Export to make unit testing easier:
```

Comment:
These 2 functions should be moved into separate files and `require`'d at the top. That way we're not bolting things onto the require that shouldn't be exposed, we improve modularity, and we can still test those 2 functions.